File: Fase 2/Backend Iteracion 2/forProject2IPC2_2023/src/main/java/com/forproject2/forproject2ipc2_2023/modelo/Examenes/Examenes.java
package com.forproject2.forproject2ipc2_2023.modelo.Examenes;

import lombok.*;

@Getter @Setter @ToString
 @NoArgsConstructor @AllArgsConstructor
@Builder
public class Examenes {
    private int id;
    private String nombre;
    private String descripcion;
    private double precio;

    public Examenes(int id, String nombre, String descripcion){
        this.id=id;
        this.nombre=nombre;
        this.descripcion=descripcion;
    }
    public Examenes(int id, double precio){
        this.id=id;
        this.precio=precio;
    }
    public Examenes(int id){
        this.id=id;
    }
}

File: Entrega Final/Frontend/proyect2IPC2/src/app/Modelo/Examenes.ts
export class Examenes {
    id!: number;
    nombre!: String;
    descripcion!: String;
    precio!: number;

    constructor(id: number,
        nombre: String,
        descripcion: String,
        precio: number){
            this.id=id;
            this.nombre=nombre;
            this.descripcion=descripcion;
            this.precio=precio;
        }
}